src/kernel/chainparams.cpp

```diff
@@ -273,6 +279,12 @@ class CTestNetParams : public CChainParams {
         base58Prefixes[EXT_PUBLIC_KEY] = {0x04, 0x35, 0x87, 0xCF};
         base58Prefixes[EXT_SECRET_KEY] = {0x04, 0x35, 0x83, 0x94};
 
+        base58EncodedPrefixes[PUBKEY_ADDRESS] = {"m","n"};
+        base58EncodedPrefixes[SCRIPT_ADDRESS] = {"2"};
+        base58EncodedPrefixes[SECRET_KEY] =     {"9","c"};
```

Comment:
Nit: I found these multiple spaces after the `=` a little unusual (first time seeing this), imo it's fine to use just one.

```diff
- base58EncodedPrefixes[SECRET_KEY] =     {"9","c"};
+ base58EncodedPrefixes[SECRET_KEY] = {"9","c"};
```